rules.css

```diff
@@ -0,0 +1,41 @@
+@import url(index.css);
+
+body {
+    background-color: var(--primary-color) ;
+    color: var(--secondary-color);
+}
+
+.instruction-container {
+    max-width: 800px;
+    margin: auto;
+    margin-top: 1rem;
+    padding: 20px;

```

Comment:
Try to use relative units (preferably rem), that way it scales as per window size, and looks consistent across different devices, screen sizes and so on.